distribution/docs/src/main/resources/_contents/_data/media-attributes-table-contents.adoc

```diff
@@ -72,18 +72,20 @@ STANAG 4559 NC, NM, C1, M1, I1, C3, M3, C4, M4, C5, M5, C8, M8
 |A two-part identifier for file formats and format content
 |String
 |a valid https://en.wikipedia.org/wiki/Media_type[mime-type]
+TODO mime-type is defined in SIX linked RFC specs, but wikipedia isn't really authoritative
 |application/json
 
 |media.encoding
 |The encoding format of the media.
-|String
-|< 1024 characters
+|List of Strings
+|< 1024 characters per entry
+TODO this needs to be more well-defined
 | 
  
 |media.number-of-bands
 |The number of spectral bands in the media.
 |Integer
-| 
+| TODO spec reference?
```

Comment:
STANAG 4545 (NSIF) defines image bands as "A well defined range of values (e.g., wavelengths, frequencies or energies of optical, electric, or acoustic radiation). At the pixel level, a band is represented as one of the vector values of the pixel. At image level band i of an image is the rectangular array of ith sample values from the pixel vectors."  

It looks like image representation (IREP) corresponds to the number of bands:
![stanag4545-table-c-1-2](https://cloud.githubusercontent.com/assets/4503545/22511177/0615625a-e852-11e6-9a4b-a4d25e7bda9e.png)

However, the spec also have "XBANDS" which are for multi-spectral images containing greater than 9 bands.  So, my opinion is that we shouldn't enumerate this and leave it as an unconstrained integer.